README.md

```diff
@@ -37,17 +37,26 @@ swiftmailer:
         spool: { type: memory }
 ```
 
-3. Implement your custom configuration class. This should implement the `Configuration\ConfigurationInterface`. For 
+4. Implement your custom configuration class. This should implement the `Configuration\ConfigurationInterface`. For
 an implementation example, have a look at `Configuration\EmptyConfiguration`. Note that the custom file is required:
 it is checked whether the `EmptyConfiguration` is still loaded, which will make this bundle throw an exception if that
 is the case.
 
-4. Register your configuration class in the kernel parameters (either `parameters.yml` or `config.yml`):
+5. Register your configuration class in the kernel parameters (either `parameters.yml` or `config.yml`):
 ```yml
 parameters:
     kickin.exceptionhandler.configuration.class: <your_class_here>
 ```
 
+6. (Optional) By default, the default configuration doesn't include access to other services in the `Configuration` class.
+Should you require these, include the new class in your own `services.yml` to decorate the default service. If you use autowiring, you're done.
+Otherwise, configure the arguments of your class here as well.
```

Comment:
There is no default configuration, just an empty configuration which throws an example [when used](https://github.com/Kick-In/exception-handler-bundle/blob/b172b57968c24dc6db8e44689348fd396d2519e8/Handler/ExceptionHandler.php#L92). 

So, the text should be something in the line of:
```
6. If you require an external service in your configuration implementation, make sure to decorate this service accordingly.
```

Reply:
I think you mean exception ;) I understand it is ambiguous as it is written now, but I meant the service configuration, not the configuration class. Will add an explanation